webrtc.html

```diff
@@ -5236,28 +5231,28 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
           object [[!GETUSERMEDIA]].</p>
         </li>
         <li>
-          <p>Initialize <var>track.kind</var> to <var>kind</var>.
+          <p>Set <var>track.kind</var> to <var>kind</var>.
         </li>
         <li>
-          <p>If an id, <var>id</var>, was given as input to this algorithm, let
-          <var>track.id</var> be <var>id</var>. (Otherwise the value generated
+          <p>If an id, <var>id</var>, was given as input to this algorithm, set
+          <var>track.id</var> to <var>id</var>. (Otherwise the value generated
```

Comment:
Definitely better, though since [id](https://w3c.github.io/mediacapture-main/getusermedia.html#widl-MediaStreamTrack-id) says: "Unless a *MediaStreamTrack* object is created as a part of a special purpose algorithm that specifies how the track id must be *initialized*, the User Agent MUST generate an identifier string and initialize the object's `id` attribute to that string."

I would think we could use "initialize" on all the track members here, including `id`, if we wanted.